index.html

```diff
@@ -644,6 +676,74 @@ <h3>
             </ol>
           </li>
         </ol>
+        <p>
+          To <dfn>record touches</dfn> for |gamepad:Gamepad|, run the following
+          steps:
+        </p>
+        <ol class="algorithm">
+          <li>Assert: {{Gamepad}}.{{Gamepad/[[touches]]}} is [=list/empty=].
+          </li>
+          <li>Repeat the following steps for each touch surface on |gamepad| in
+          touch surface enumeration order:
+            <ol>
+              <li>Let |surfaceId:unsigned long| be the current surface
+              enumeration index.
+              </li>
+              <li>If the touch surface exposes maximum surface dimensions in
+              device units, then set |touch|.{{GamepadTouch/surfaceDimensions}}
+              to a {{DOMRect}} with `width` and `height` initialized to the
+              maximum X and Y dimensions on the touch surface in device units.
+              </li>
+              <li>Repeat the following steps for each active touch point
+              reported by the |gamepad| for the current touch surface.
+                <ol>
+                  <li>Let |nextTouchId:unsigned long| be the next available
+                  touchId for the |gamepad|.
+                  </li>
+                  <li>Let |touch:GamepadTouch| be a newly created
+                  {{GamepadTouch}} object.
+                  </li>
+                  <li>Set |touch|.{{GamepadTouch/surfaceId}} to be |surfaceId|.
+                  </li>
+                  <li>If the touch data is part of an existing active touch
+                  point tracked by the user agent:
+                    <ol>
+                      <li>Set |touch|.{{GamepadTouch/touchId}} to the touchId
+                      of the active touch point.
+                      </li>
+                      <li>Otherwise, set touchId to nextTouchId and increment
+                      nextTouchId.
+                        <p class="note" title=
+                        "Touch ids are relative to the Gamepad">
+                          If the Gamepad has multiple touch surfaces the touch
+                          id will be unique across surfaces.
+                        </p>
+                      </li>
+                    </ol>
+                  </li>
+                  <li>Let |position| be a [=new=] {{DOMPoint}} with `x`
+                  initialized to device X coordinate relative to the device
+                  touch surface and normalized to [-1.0,1.0] where -1.0 is the
+                  leftmost coordinate and 1.0 is the rightmost coordinate and
+                  `y` initialized to the device touch surface and normalized to
+                  [-1.0,1.0] where -1.0 is the leftmost coordinate and 1.0 is
+                  the rightmost coordinate.
```

Comment:
`|position|` is declared and not used. Should we write something like "Set |touch|.{{GamepadTouch/position}} to |position|"?